src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/utils/ApiLoggerTest.java

```diff
@@ -41,7 +41,7 @@ void testInfoLoggingDoesNotModifyLogMap() {
 
     @Test
     void testInfoContextLoggingDoesNotModifyLogMap() {
-        ApiLogger.infoContext(CONTEXT, TEST_MESSAGE);
+        ApiLogger.infoContext(CONTEXT, TEST_MESSAGE, logMap);
```

Comment:
Is it worth testing debug and error logging here as well?

Reply:
i haven't changed those, only added a new infoContext method

Reply:
Ignore, we seem to have those.